x/stakeibc/keeper/msg_server_redeem_stake.go

```diff
@@ -42,7 +42,7 @@ func (k msgServer) RedeemStake(goCtx context.Context, msg *types.MsgRedeemStake)
 
 	// construct desired unstaking amount from host zone
 	coinDenom := "st" + hostZone.HostDenom
-	nativeAmount := sdk.NewDec(msg.Amount).Mul(hostZone.RedemptionRate)
+	nativeAmount := sdk.NewDec(cast.ToInt64(msg.Amount)).Mul(hostZone.RedemptionRate)
```

Comment:
flagging

Reply:
Addressed per new cast fork and using cast.To____E